nextflow_schema.json

```
@@ -41,46 +73,148 @@
+                    "fa_icon": "fab fa-cuttlefish"
                 },
-                "igenomes_ignore": {
+                "save_trimmed": {
                     "type": "boolean",
-                    "description": "Do not load the iGenomes reference config.",
-                    "fa_icon": "fas fa-ban",
-                    "hidden": true,
-                    "help_text": "Do not load `igenomes.config` when running the pipeline. You may choose this option if you observe clashes between custom parameters and those supplied in `igenomes.config`."
+                    "description": "Save the trimmed FastQ files in the results directory.",
+                    "fa_icon": "fab fa-cuttlefish"
+                },
+                "skip_trimming": {
+                    "type": "boolean",
+                    "description": "Skip the adapter trimming step.",
+                    "fa_icon": "fas fa-forward"
                 }
-            }
+            },
+            "fa_icon": "fas fa-cut"
+        },
+        "BBtools_options": {
+            "title": "BBtools options",
+            "type": "object",
+            "description": "All the options you can use for BBmap or BBduk)",
+            "default": "",
+            "properties": {
+                "sequence_filter": {
+                    "type": "string",
+                    "description": "Instructs BBduk to use a fasta file to filter away sequences before running further analysis.",
+                    "fa_icon": "fas fa-filter"
+                },
+                "bbmap_minid": {
+                    "type": "number",
+                    "default": 0.9,
+                    "description": "Minimal identity for BBmap",
+                    "fa_icon": "fas fa-search-minus"
+                },
+                "save_bam": {
+                    "type": "boolean",
+                    "description": "Save bam output file",
+                    "fa_icon": "fas fa-save"
+                },
+                "save_bbmap_index": {
+                    "type": "boolean",
+                    "description": "Save ref folder containing the reference index"
+                },
+                "save_genomes_cat": {
+                    "type": "boolean",
+                    "description": "Save genomes concatenated file"
+                }
+            },
+            "fa_icon": "fas fa-align-justify"
+        },
+        "prokka_option": {
+            "title": "Prokka option",
+            "type": "object",
+            "description": "Prokka option",
+            "default": "",
+            "properties": {
+                "save_prokka": {
+                    "type": "boolean",
+                    "fa_icon": "fas fa-file-download",
+                    "description": "Enable to save prokka output. Default: false"
+                }
+            },
+            "fa_icon": "fas fa-phone-alt"
+        },
+        "Sourmash_options": {
```

```suggestion
        "sourmash_options": {
```

Shouldn't it be all lower case?